GithubJobApp/GithubApiManager.swift

```diff
@@ -10,60 +10,44 @@ import Foundation
 
 struct GithubApiManager {
     
-    let jobsUrl: String = "https://jobs.github.com/positions.json?description=swift"
+    private let jobsUrl: String = "https://jobs.github.com/positions.json?description=swift"
     
-    // Dont need these vars
-//    var jobTitles = [String]()
-//    var jobCompanies = [String]()
-    
-    func getJobs(setDataFunction: (Array<Job>) -> Void){
+    func getJobs(completion: ([Job]) -> Void) {
         if let url = NSURL(string: jobsUrl) {
-        let task = NSURLSession.sharedSession().dataTaskWithURL(url) {(data, response, error) in
-            if let actualData = data { print(NSString(data: actualData, encoding: NSUTF8StringEncoding)) }
-            
-            var jobArray = [Job]()
-            var json: Array<AnyObject>?
-            do {
-                guard let actualData = data else {
+            let task = NSURLSession.sharedSession().dataTaskWithURL(url) {(data, response, error) in
+                guard let jobs = data else {
                     print("error: data recieved is invalid")
```

Comment:
Just FYI, if the data returned is nil, then no data was returned, not necessarily invalid data.